content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -51,8 +51,8 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 
 ### Removed
```

Comment:
In #656, @gluszczyk proposed a really neat format for our Changelogs. What about we change the Removed section as follows:

> ### Removed
> 
> **Methods:**
> - Update Agent
> - Get Chat Threads Summary
> 
> **Pushes:**
> - `agent_updated`

Reply:
I'll clean up other Changelogs before v3.2 becomes stable